build.sh

```diff
@@ -21,24 +23,78 @@ OL_UBI_IMAGE="${OL_UBI_IMAGE:-openliberty/open-liberty:20.0.0.10-kernel-java11-o
 #
 # The name and tag of the "stack image you will build.  This will used to create your inner loop development containers, and also as the base image for the first stage of your outer loop image build.
```

Comment:
This is irrelevant for this PR, but perhaps we want to start this 4.0 release with 20.0.0.11.

Reply:
Yeah, went back and forth on this.   

My thought is that the "gateway" into our stuff at the moment is each of:
* odo create
* CP4Apps stack yaml
and both point to the Devfile registry.

So, since this is a big change with a lot of moving parts... I was thinking let's release the existing function at 0.4 with this new release method.   But then release over it, even over 0.4.0, with the 20.0.0.11 kernel-slim change.